bin/antigen.zsh

```diff
@@ -1402,7 +1402,7 @@ antigen-use () {
 antigen-version () {
```

Comment:
I have a question. I'm not sure to understand the whole solution and its deployment mechanisms. But shouldn't this function be deleted as it seems redundant with the one in `src/commands/version.zsh`

Reply:
I'm not sure; I'd prefer to keep it for now and remove it in a separate PR (since I too don't really understand how it's relevant).